src/main/java/com/google/sps/servlets/MentorApprovalServlet.java

```diff
@@ -47,15 +47,15 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     int userId = Utility.getUserId();
     
     // Set default variables to create MentorEvidence object.
-    boolean isApprover = false;
     String mentorUsername = "";
     boolean isApproved = false;
     boolean isRejected = false;
     String paragraph = "";
+    boolean[] approver = {false, false}; // Stores approver ID and if they have reviewed already.
```

Comment:
Is this really storing an ID? Perhaps changing the wording of the comment will help clarify.

Reply:
Don't know why I wrote that. Changed it!